astro/src/content/docs/lifecycle/authenticate-users/oauth/endpoints.mdx

```diff
@@ -319,6 +319,8 @@ Location: https://piedpiper.com/callback#
 
```

Comment:
Might make sense to extract this entire user state description to an include file since it looks the same.